plaso/engine/artifact_filters.py

```diff
@@ -0,0 +1,239 @@
+# -*- coding: utf-8 -*-
+"""Artifact filters processing."""
+
+from __future__ import unicode_literals
+
+import logging
+import re
+
+from artifacts import definitions as artifact_types
+
+from dfvfs.helpers import file_system_searcher
+from dfwinreg import registry_searcher
+
+from plaso.engine import path_helper
+
+
+ARTIFACT_FILTERS = 'ARTIFACT_FILTERS'
+COMPATIBLE_DFWINREG_KEYS = ['HKEY_LOCAL_MACHINE',
+                            'HKEY_LOCAL_MACHINE\\SYSTEM',
+                            'HKEY_LOCAL_MACHINE\\SOFTWARE',
+                            'HKEY_LOCAL_MACHINE\\SAM',
+                            'HKEY_LOCAL_MACHINE\\SECURITY']
+STANDARD_OS_FILTERS = ['windows',
+                       'linux',
+                       'darwin']
+RECURSIVE_GLOB_LIMIT = 10
+
+
+class ArtifactFilters(object):
+  """Forensic artifacts filters.
+
+  Build extraction and parsing filters from forensic artifact definitions.
```

Comment:
I opt to remove  "and parsing" seeing at this point we only support to create path filters

Reply:
Merged in #1883